.travis.yml

```diff
@@ -0,0 +1,20 @@
+os:
+  - linux
+  - osx
+
+env:
+  - TEST_CONNECTION=lcl TEST_SHELL=bash
+  - TEST_CONNECTION=lcl TEST_SHELL=zsh
+
+before_install:
+  - |
+    if [[ "$TRAVIS_OS_NAME" == osx ]]; then
+      brew install bats zsh
+    else
+      sudo add-apt-repository ppa:duggan/bats --yes
+      sudo apt-get update -qq
+      sudo apt-get install -qq -y bats zsh
```

Comment:
I've done this, but it appears to be a beta feature on travis and travis is not happy: https://travis-ci.org/felixSchl/liquidprompt/jobs/162850367